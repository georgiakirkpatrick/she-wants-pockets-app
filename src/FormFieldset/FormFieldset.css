* {
    box-sizing: border-box;
    padding: 0;
    margin: 0;
}

.FormFieldset {
    display: flex;
    flex-direction: column;
    align-items: flex-start;
    margin: 10px 0;
}