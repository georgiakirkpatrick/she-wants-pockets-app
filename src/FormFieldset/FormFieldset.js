import React from 'react'
import './FormFieldset.css'

const FormFieldset = props => {
    const prompt = props.prompt ?  <p className='FormPromptWithSub_prompt'>{props.prompt}</p> : <div className='empty' />

    return (
        <fieldset className='FormFieldset'>
            {prompt}
            {props.children}
        </fieldset>
    )
}

export default FormFieldset