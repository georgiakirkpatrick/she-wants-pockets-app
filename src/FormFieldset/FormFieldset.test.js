import React from 'react'
import ReactDOM from 'react-dom'
import renderer from 'react-test-renderer'
import FormFieldset from './FormFieldset'

describe('FormFieldset', () => {
    it ('renders without crashing', () => {
        const div = document.createElement('div')
        ReactDOM.render(<FormFieldset />, div)
        ReactDOM.unmountComponentAtNode(div)
    })

    it ('render the UI as expected', () => {
        const tree = renderer
            .create(<FormFieldset />)
            .toJSON()
        expect(tree).toMatchSnapshot()
    })
})