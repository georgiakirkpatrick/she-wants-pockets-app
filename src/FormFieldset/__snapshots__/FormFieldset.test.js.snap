// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FormFieldset render the UI as expected 1`] = `
<fieldset
  className="FormFieldset"
>
  <legend />
</fieldset>
`;
